ethexe/observer/src/tests.rs

```diff
@@ -50,7 +55,31 @@ async fn test_deployment() -> Result<()> {
     let sender_address = sender_public_key.to_address();
     let validators = vec!["0x45D6536E3D4AdC8f4e13c5c4aA54bE968C55Abf1".parse()?];
 
-    let ethereum = Ethereum::deploy(&ethereum_rpc, validators, signer, sender_address).await?;
+    let (secret_shares, _) = keys::generate_with_dealer(
+        1,
+        1,
+        IdentifierList::Custom(&[Identifier::deserialize(
+            &ActorId::from(validators[0]).into_bytes(),
+        )
+        .unwrap()]),
+        rand::thread_rng(),
+    )
+    .unwrap();
+
+    let verifiable_secret_sharing_commitment = secret_shares
+        .values()
+        .map(|secret_share| secret_share.commitment().clone())
+        .next()
+        .expect("conversion failed");
```

Comment:
mb wrap into fn somewhere for future usage ?

Reply:
not now, ok?